src/Illuminate/Support/Arr.php

```diff
@@ -342,6 +342,30 @@ public static function has($array, $keys)
         return true;
     }
 
+    /**
+     * Check if an item or items are filled in an array using "dot" notation.
+     *
+     * @param  \ArrayAccess|array  $array
+     * @param  string|array  $keys
+     * @return bool
+     */
+    public static function filled($array, $keys)
+    {
+        if (! static::has($array, $keys)) {
+            return false;
+        }
+
+        foreach ((array) $keys as $key) {
+            $needle = static::get($array, $key);
+
+            if (empty($needle)) {
```

Comment:
You should use the `blank()` helper here.
The idea of something "filled" could be a bit confusing if we use the `empty()` function as `empty(collect())` returns `false` and `empty([])` returns `true` but both are empty "lists"

Reply:
Some more use cases:

```php
empty(0);
// true

empty('0');
// true

empty(false);
// true
```

According to Laravel's `Request::filled()` and the `filled()` global helper, these three examples are considered "present" values, not blank.